### DISCUSSION TITLE
Guide -> General Setup -> Run admin/setup

### DISCUSSION THREAD

**Discussion by ssadguru** [luyadev/luya#2161] [Questions & Problems]
During the admin/setup I am getting following error...

**[bhakticlasses@titano luya-kickstarter]$** ./vendor/bin/luya admin/setup
**User E-Mail:** s_sadguru@yahoo.com
**User Password:** Ss_140975
**Firstname:** Santosh
**Lastname:** Sadguru
**Standard language: [English]**
**Short-Code of the Standard language: [en]**
**E-Mail: s_sadguru@yahoo.com
Firstname: Santosh
Lastname: Sadguru
Language: English
Confirm your login details in order to proceed with the Setup. Are those informations correct? (yes|no) [no]:**yes
**Exception 'yii\db\Exception' with message 'SQLSTATE[HY000]: General error: 2006 MySQL server has gone away
The SQL being executed was: SHOW FULL COLUMNS FROM `admin_user`'

in /home/bhakticlasses/public_html/motherscholarship.vedsoft.co.in/luya-kickstarter/vendor/yiisoft/yii2/db/Schema.php:676

Error Info:
Array
(
    [0] => HY000
    [1] => 2006
    [2] => MySQL server has gone away
)

Caused by: Exception 'PDOException' with message 'SQLSTATE[HY000]: General error: 2006 MySQL server has gone away'

in /home/bhakticlasses/public_html/motherscholarship.vedsoft.co.in/luya-kickstarter/vendor/yiisoft/yii2/db/Command.php:1302**

Please guide me how to solve the error ?

**Comment by nadar** [comment]
hi @ssadguru 

This looks like you can not connect to the sql instance. Did you run `./vendor/bin/luya migrate` successful? Are you using docker-compose? Or can you tell us more about your setup?
